CorsixTH/Lua/languages/dutch.lua

```diff
@@ -629,31 +629,6 @@ fax = {
   vip_visit_result = {
     telegram = "Telegram!",
     remarks = {
-      good = {
-        [1] = "Dat ziekenhuis draait goed. Bedankt voor de uitnodiging.",
-        [2] = "Hmm. Zeker een niet al te kwade medische instelling.",
-        [3] = "Ik heb genoten van je charmante ziekenhuis. Goed, iemand zin in Nasi Goreng van de Indonees?",
-      },
-      super = {
-        [1] = "Dat ziekenhuis loopt als een trein. Als mijn gezondheid me in de steek laat, wil ik daarnaartoe.",
-        [2] = "Dat is pas een ziekenhuis!",
-        [3] = "Wat een fantastisch ziekenhuis. En ik kan het weten: ik heb er wel een paar van binnen gezien.",
-      },
-      bad = {
-        [1] = "Was dat het nou? Het was nog erger dan een opera van vier uur!",
-        [2] = "Ik walg van wat ik gezien heb. Was dat een ziekenhuis? Het leek meer op een zwijnenstal!",
-        [3] = "Ik ben het zat een publiek figuur te zijn en zulke ranzige hokken te moeten bezoeken! Ik kap ermee.",
-      },
-      mediocre = {
-        [1] = "Nou ja, het kan erger. Maar je moet er wat aan werken.",
-        [2] = "Oh jee. Niet zo een fijne plaats als je je niet lekker voelt.",
-        [3] = "Een matig ziekenhuis, eerlijk gezegd. Ik had meer verwacht.",
-      },
-      very_bad = {
-        [1] = "Wat een zooi. Ik ga kijken of ik het kan laten sluiten.",
-        [2] = "Ik heb nog nooit zo'n verschrikkelijk ziekenhuis gezien. Wat een schande!",
-        [3] = "Ik ben geschokt. Dat is toch geen ziekenhuis? Ik heb een biertje nodig.",
```

Comment:
@Alberth289346 I'm not sure if you wanted to collab with me on remapping these to the original strings' `fax.vip_visit_result.remarks[1 - 12]` instead of me just removing them?

Other languages are accommodated for.

Reply:
Ah, reading it again, I understand. Let's see :)